CONTRIBUTING.md

```diff
@@ -28,6 +28,11 @@ Your Pull Request **must** always pass the Travis/Appveyor tests before being me
 
 _If you are adding a dependency (python module, library, ...), please check the [corresponding section](#add-dependencies)._
 
+## [Integrations](https://github.com/DataDog/integrations-core)
+
+All integrations, except for the kubernetes and docker ones, have been moved to the [Integration SDK](https://github.com/DataDog/integrations-core). Please look there to contribute to them.
```

Comment:
Oh I need to update that as well, we have the deprecated hdfs, jenkins and mesos checks in here too, along with the agent_metrics check.

We are but not before the release. Currently they are too tied into the core agent to remove.